tenant/config/crds/tenant_v1alpha1_tenant.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  creationTimestamp: null
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: tenants.tenant.x-k8s.io
+spec:
+  group: tenant.x-k8s.io
```

Comment:
Groups are often plural if it's a concrete noun (eg "apps", "extensions") or singular if it's a concept (eg "policy," "authentication," "networking"). Should this be "tenancy.x-k8s.io" or "tenants.x-k8s.io" instead?

Reply:
Other than this, lgtm

Reply:
I kind of like tenancy.